src/vm/stackwalk.cpp

```diff
@@ -725,7 +723,7 @@ PCODE Thread::VirtualUnwindNonLeafCallFrame(T_CONTEXT* pContext, KNONVOLATILE_CO
 #if defined(_WIN64)
     UINT64              EstablisherFrame;
```

Comment:
@janvorli Is ``_WIN64`` is for general 64-bit architecture? Then, it would be better to use _BIT64 instead of _WIN64 (and _BIT32 for the below as well).